Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -17,20 +17,16 @@ open class _ChatChannelListItemView<ExtraData: ExtraDataTypes>: _View, UIConfigP
 
     /// The date formatter of the `timestampLabel`
     public lazy var dateFormatter: DateFormatter = .makeDefault()
-    
-    /// Properties tied to `ChatChannelListItemView` layout
-    public struct Layout {
-        /// Constraints of `timestampLabel`
-        public fileprivate(set) var timestampLabelConstraints: [NSLayoutConstraint] = []
-        /// Constraints of `avatarView`
-        public fileprivate(set) var avatarViewConstraints: [NSLayoutConstraint] = []
-        /// Constraints of `titleLabel`
-        public fileprivate(set) var titleLabelConstraints: [NSLayoutConstraint] = []
-        /// Constraints of `subtitleLabel`
-        public fileprivate(set) var subtitleLabelConstraints: [NSLayoutConstraint] = []
-        /// Constraints of `unreadCountView`
-        public fileprivate(set) var unreadCountViewConstraints: [NSLayoutConstraint] = []
-    }
+
+    /// Main container which holds `avatarView` and two horizontal containers `title` and `unreadCount` and
+    /// `subtitle` and `timestampLabel`
+    open private(set) lazy var mainContainer: ContainerView = ContainerView()
+
+    /// By default contains `title` and `unreadCount`.
+    open private(set) lazy var topContainer: ContainerView = ContainerView()
```

Comment:
I'd maybe add here a comment about the fact that this is above `mainContainer`. Although it has `top` in the name, it might not be immediately obvious in relation to which element it is "top". Same for `bottomContainer`